### PR TITLE
Three new commands

### PR DESCRIPTION
**Pull request by ChrisPrusik**
getinnerhtml
getouterhtml
gethtml

### CODE REVIEW

**Comment by ChrisPrusik** [comment]
[AB#622](https://dev.azure.com/g1antdevops/4b953fde-dbe6-4a8c-a404-fbe9df3338e9/_workitems/edit/622)

File: G1ANT.Addon.Selenium/Commands/SeleniumGetIOuterHtml.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/**
```
Comment (pwitusik) [inline G1ANT.Addon.Selenium/Commands/SeleniumGetIOuterHtml.cs]: raname the file SeleniumGetIOuterHtml.cs -> SeleniumGetOuterHtml.cs

File: G1ANT.Addon.Selenium/Api/SeleniumIFrameArguments.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using G1ANT.Language;
+
+namespace G1ANT.Addon.Selenium
+{
+    public class SeleniumIFrameArguments : CommandArguments
```
Comment (wojszub) [inline G1ANT.Addon.Selenium/Api/SeleniumIFrameArguments.cs:5]: you should change SeleniumCommandArguments base class to SeleniumIFrameArguments and remove redundant properties

File: G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs [hunk 3]
```diff
@@ -364,6 +364,42 @@ public string GetAttributeValue(string attributeName, SeleniumCommandArguments s
...
+            PreCheckCurrentWindowHandle();
+            if (!string.IsNullOrEmpty(search.IFrameSearch?.Value))
+                webDriver.SwitchTo().Frame(FindElement(search.IFrameSearch.Value, search.IFrameBy.Value, timeout));
+            var element = FindElement(search.Search.Value, search.By.Value, timeout);
+            var res = element?.GetAttribute("outerHTML") ?? string.Empty;
+            if (!string.IsNullOrEmpty(search.IFrameSearch?.Value))
+                webDriver.SwitchTo().DefaultContent();
+            return res;
+        }
+
```
Comment (wojszub) [inline G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs]: this methods looks like DRY violation, BTW you can use GetAttributeValue() method without creating new ones

File: G1ANT.Addon.Selenium/Commands/SeleniumGetHtml.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
+/**
+*    Copyright(C) G1ANT Ltd, All rights reserved
```
Comment (wojszub) [inline G1ANT.Addon.Selenium/Commands/SeleniumGetHtml.cs:2]: shouldn't be "G1ANT Robot Ltd"?

**Review by wojszub — CHANGES_REQUESTED** [review]
we should follow some branching rules (branch name) [Wiki](https://dev.azure.com/g1antdevops/G1ANT.Robot/_wiki/wikis/G1ANT.Robot.wiki/20/Developer's-handbook?anchor=branching-strategy)

File: G1ANT.Addon.Selenium/Api/SeleniumIFrameArguments.cs (hunk 2 above)
Comment (ChrisPrusik) [reply G1ANT.Addon.Selenium/Api/SeleniumIFrameArguments.cs:5]: zrobiłem tak kolejność jest ok. Zaraż zrobię pusha

File: G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs (hunk 3 above)
Comment (ChrisPrusik) [reply G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs]: Jasne

File: G1ANT.Addon.Selenium/Commands/SeleniumGetHtml.cs (hunk 4 above)
Comment (ChrisPrusik) [reply G1ANT.Addon.Selenium/Commands/SeleniumGetHtml.cs:2]: Should be changing everywhere. We need decision from Business how to do this.

**Comment by ChrisPrusik** [comment]
Thanks @wojszub!

**Review by wojszub — APPROVED** [review]

**Review by pwitusik — APPROVED** [review]

**Pull request merged**